### PR TITLE
Add WeTransfer

### LINKED ISSUE #7267: Add WeTransfer
### Site name

Add WeTransfer

### Site URL

https://wetransfer.com/

### Documentation URL

https://help.wetransfer.com/hc/en-us/articles/360056666872

### Recovery Documentation URL

https://help.wetransfer.com/hc/en-us/articles/360056666872-Enable-two-factor-authentication-on-your-WeTransfer-account#h_01EXEFX7G5DKD2GHYDGWH1S0C7

### Supported 2FA methods

- [X] TOTP (RFC-6238, Google Authenticator)
- [ ] Proprietary Software (Authy, Duo, etc.)
- [ ] U2F/WebAuthn security keys
- [ ] Proprietary hardware (Yubico OTP, RSA keys, etc.)
- [ ] Phone calls
- [ ] SMS tokens
- [ ] Email tokens

### Regions served

_No response_

### Eligibility

- [X] I have checked that the site meets 2fa.directory's [contributing guidelines and site criteria](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md)
- [X] The site is ranked within the top 200,000 sites on [Similarweb](https://www.similarweb.com/).
- [X] The site does not contain or promote content that violates 2fa.directory's [excluded categories and websites guidelines](https://github.com/2factorauth/twofactorauth/blob/master/EXCLUSION.md).
- [X] The issue I'm creating is not a duplicate of an existing [issue](https://github.com/2factorauth/twofactorauth/issues) or [pull request](https://github.com/2factorauth/twofactorauth/pulls).

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by hkamran80**
Resolves #7267

### CODE REVIEW

File: entries/w/wetransfer.com.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+        "domain": "wetransfer.com",
+        "tfa": [
+            "totp"
+        ],
+        "documentation": "https://help.wetransfer.com/hc/en-us/articles/360056666872",
+        "recovery": "https://help.wetransfer.com/hc/en-us/articles/360056666872#h_01EXEFX7G5DKD2GHYDGWH1S0C7",
+        "categories": [
+            "creativity",
+            "social",
+            "other"
```
Comment (Carlgo11) [inline entries/w/wetransfer.com.json]: So it's a file uploading site?
The __Sync__ in Backup & Sync makes that category a better fit for these types of sites.
```suggestion
            "backup"
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: entries/w/wetransfer.com.json (hunk 1 above)
Comment (hkamran80) [reply entries/w/wetransfer.com.json]: They have [other products](https://wetransfer.com/explore/products) too, specifically [Paste](https://wetransfer.com/paste), which shares a common authentication backend with WeTransfer.

Comment (Carlgo11) [reply entries/w/wetransfer.com.json]: Okay so backup and creativity?

Comment (hkamran80) [reply entries/w/wetransfer.com.json]: Yup. I'll update the PR.

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**